### PR TITLE
[Test only]

### PR DESCRIPTION
**Pull request by RussKie**
I copied the content from eng/pipelines/azure-pipelines-public.yml into eng/pipelines/azure-pipelines.yml, and it it built successfully. Is there anything else you can think of to test?
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/2801)

### CODE REVIEW

File: eng/pipelines/azure-pipelines-public.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ variables:
     value: ''
   - name: _Sign
     value: false
+  - name: HelixApiAccessToken
```
Comment (radical) [inline eng/pipelines/azure-pipelines-public.yml:39]: This looks good 👍

File: eng/pipelines/azure-pipelines.yml [hunk 2]
```diff
@@ -121,47 +91,41 @@ stages:
         steps:
           - template: /eng/pipelines/templates/BuildAndTest.yml
             parameters:
-              runAsPublic: ${{ eq(variables._RunAsPublic, True) }}
+              runAsPublic: true
```
Comment (radical) [inline eng/pipelines/azure-pipelines.yml:94]: Is this file now used only for the internal pipeline? If yes, then should this be false?

Comment (radical) [reply eng/pipelines/azure-pipelines.yml:94]: oh, this is what you want for the public one and are testing in this file :D

File: eng/pipelines/azure-pipelines-public.yml (hunk 1 above)
Comment (joperezr) [reply eng/pipelines/azure-pipelines-public.yml:39]: Yup, looks like this fixed the issue.

**Comment by joperezr** [comment]
This LGTM, thanks for being thorough @RussKie. I don't think that we need any other validation, so I would expect that undoing the changes in azure-pipelines, and checking in the public change one should now allow us to switch the pipeline definition.

File: eng/pipelines/azure-pipelines.yml (hunk 2 above)
Comment (RussKie) [reply eng/pipelines/azure-pipelines.yml:94]: Yes, this is `true` for the public pipelines only.

**Pull request closed without merging**